lecture-7/Book_author/Book.rb

```diff
@@ -0,0 +1,46 @@
+
+class Book
+  attr_reader :name, :authors, :issued, :year
+  @@pattern_name = /[\p{L}\d\-.,;]{1,10}/
+  @@pattern_year = /\d{4}/
+
+  def initialize(name, authors, issued, year)
```

Comment:
refactor this method without authors_pass flag

Reply:
done using  @authors = nil